tests/jeos/firstrun.pm

```diff
@@ -111,7 +111,7 @@ sub verify_partition_label {
 
     # The RPi firmware needs MBR. s390x images also use MBR.
     # Note: JeOS-for-RaspberryPi means "kiwi-templates-Minimal" and JeOS-for-RPi means "community JeOS".
```

Comment:
As you prefer, comment update to describe new expression too, could be useful

Reply:
added :)